src/random.cr

```diff
@@ -1,66 +1,125 @@
 require "random/mt19937"
 
+# `Random` provides an interface for random values generation, using a pseudo random number generator (PRNG).
+# 
+# ```
+# Random.new_seed # => 112705036
+# Random.rand     # => 0.167595
+# Random.rand(5)  # => 2
+# ```
+#
+# The above methods delegate to a shared instance. You can create one like:
+#
+# ```
+# r = Random.new
+# r.next_bool # => true
+# r.next_int  # => 2223112
+# ```
+#
+# It also defines a global method `#rand`, which `Array#sample` delegates.
```

Comment:
Now it sounds like "It" is referring to the shared method implementation. What about "The module also defines a ..."?